tests/dummy_tesseract/tesseract_api.py

```diff
@@ -41,6 +43,10 @@ def apply(inputs: InputSchema) -> OutputSchema:
         norm = np.linalg.norm(result, ord=2)
         result /= norm
 
+    env_var = os.getenv("TEST_ENV_VAR")
+    if env_var:
+        print(f"TEST_ENV_VAR: {env_var}")
```

Comment:
In cases like this we usually apply the following trick, so you don't have to manipulate the dummy Tesseract:

1. Call `tesseract serve` (here with `--env`) to create a running container.
2. Grab the container ID via the Docker client.
3. Call `container.exec_run("echo $TEST_ENV_VAR")` or similar to execute commands against the running Tesseract.

Reply:
Nice! That works well for the `serve` case.
I need to make the same work for `run` though, since both have different code paths for handling CLI arguments. Is there a similar way to "inject" code in that case, even though the container doesn't persist?

Reply:
Not really, but leaving that untested should be acceptable for now.